### ISSUE TITLE
Slope definition in WPS script 

### ISSUE THREAD

**Issue by ebocher** [Ifsttar/NoiseModelling#339]
The slope definition available here https://github.com/Ifsttar/NoiseModelling/blob/cnossos/wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/NoiseModelling/Noise_level_from_traffic.groovy#L83 says "If the field is not filled in, the LINESTRING z-values will be used to calculate..."
What about if the LINESTRING has a 2 coordinate dimension ? Are the  z ordinates initialized to 0?

**Comment by pierromond** [comment]
Yes... if not filled in and if not z-values... no slope. Do you think we should add sthg ?

**Comment by ebocher** [comment]
A message must be added to  said that if the linestring is 2D then the linestring is converted to 3D with a z values equal to zero.
